ext/opcache/jit/zend_jit.h

```diff
@@ -162,7 +162,8 @@ void zend_jit_startup(void *jit_buffer, size_t size, bool reattached);
 void zend_jit_shutdown(void);
 void zend_jit_activate(void);
 void zend_jit_deactivate(void);
-void zend_jit_status(zval *ret);
+ZEND_EXT_API void zend_jit_status(zval *ret);
+ZEND_EXT_API void zend_jit_blacklist_function(zend_op_array *op_array);
```

Comment:
Make a normal API not a yet another hack for your observer.

Reply:
I don't understand what you mean? What would "a normal API" look like?